### DISCUSSION TITLE
New demo release 11.15.0

### DISCUSSION THREAD

**Discussion by palexdev** [palexdev/MaterialFX#298] [General]
[Changelog](https://github.com/palexdev/MaterialFX/blob/main/CHANGELOG.md)

<hr /><em>This discussion was created from the release <a href='https://github.com/palexdev/MaterialFX/releases/tag/v11.15.0'>New demo release 11.15.0</a>.</em>

**Comment by palexdev** [comment]
Please be patient, from the current laptop I cannot push the new version on Maven. It will be available in a few hours,

**Comment by Tacoo99** [comment]
Okay, i'll wait. 
Thank you and keep the good work.
